engine/src/test/java/com/datatorrent/stram/StramMiniClusterTest.java

```diff
@@ -227,6 +242,7 @@ public void testSetupShutdown() throws Exception
   private LogicalPlan createDAG(LogicalPlanConfiguration lpc) throws Exception
   {
     LogicalPlan dag = new LogicalPlan();
+    addExtraTestJar(dag);
```

Comment:
Why this function is called for every createDAG? What functionality does it add? Should not this be just one of test cases?